File: templates/Uwp/Services/SampleData._VB/Param_ProjectName.Core/Models/SampleCompany.vb
Namespace Models
    ' TODO WTS: Remove this class once your pages/features are using your data.
    ' This is used by the SampleDataService.
    ' It is the model class we use to display data on pages like Grid, Chart, and List Detail.
    Public Class SampleCompany
        Public Property CompanyID As String

        Public Property CompanyName As String

        Public Property ContactName As String

        Public Property ContactTitle As String

        Public Property Address As String

        Public Property City As String

        Public Property PostalCode As String

        Public Property Country As String

        Public Property Phone As String

        Public Property Fax As String

        Public Property Orders As ICollection(Of SampleOrder)
    End Class
End Namespace

File: code/TemplateStudioForWinUICs/Templates/Ft/SampleData/Param_ProjectName.Core/Models/SampleCompany.cs
﻿using System.Collections.Generic;

namespace Param_RootNamespace.Core.Models
{
    // Model for the SampleDataService. Replace with your own model.
    public class SampleCompany
    {
        public string CompanyID
        {
            get; set;
        }

        public string CompanyName
        {
            get; set;
        }

        public string ContactName
        {
            get; set;
        }

        public string ContactTitle
        {
            get; set;
        }

        public string Address
        {
            get; set;
        }

        public string City
        {
            get; set;
        }

        public string PostalCode
        {
            get; set;
        }

        public string Country
        {
            get; set;
        }

        public string Phone
        {
            get; set;
        }

        public string Fax
        {
            get; set;
        }

        public ICollection<SampleOrder> Orders
        {
            get; set;
        }
    }
}
